src/cart/crt_context.c

```diff
@@ -172,8 +172,8 @@ crt_context_init(crt_context_t crt_ctx)
 	return rc;
 }
 
-int
-crt_context_provider_create(crt_context_t *crt_ctx, int provider)
+static int
+crt_context_provider_create(crt_context_t *crt_ctx, int provider, int swim_idx)
```

Comment:
this is meant to be public api function for multiprovder support. You likely want to use/create some new internal function instead with swim_idx being passed into it.

Reply:
ok. will undo changes.